### PR TITLE
cmd/repo-updater: Add repo permissions sync grafana panel

### PR DESCRIPTION
**Pull request by sashaostrikov**
Adding grafana panel for monitoring which is created in https://github.com/sourcegraph/sourcegraph/pull/28241

Closes https://github.com/sourcegraph/sourcegraph/issues/27790

### CODE REVIEW

File: monitoring/definitions/repo_updater.go [hunk 1]
```diff
@@ -301,6 +301,15 @@ func RepoUpdater() *monitoring.Container {
...
 							`,
 						},
+						{
+							Name:           "perms_syncer_scheduled_repos_total",
+							Description:    "total number of repos scheduled for permissions sync",
+							Query:          `max(rate(src_repoupdater_perms_syncer_schedule_repos_total[1m]))`,
+							NoAlert:        true,
+							Panel:          monitoring.Panel().Unit(monitoring.Number),
+							Owner:          monitoring.ObservableOwnerCoreApplication,
+							Interpretation: "Indicates how many repositories have been scheduled a permissions sync",
```
Comment (ryanslade) [inline monitoring/definitions/repo_updater.go:311]:
```suggestion
							Interpretation: "Indicates how many repositories have been scheduled for a permissions sync",
```

**Review by ryanslade — APPROVED** [review]
LGTM

File: monitoring/definitions/repo_updater.go [hunk 2]
```diff
@@ -301,6 +301,17 @@ func RepoUpdater() *monitoring.Container {
...
+						{
+							Name:        "perms_syncer_scheduled_repos_total",
+							Description: "total number of repos scheduled for permissions sync",
+							Query:       `max(rate(src_repoupdater_perms_syncer_schedule_repos_total[1m]))`,
+							NoAlert:     true,
+							Panel:       monitoring.Panel().Unit(monitoring.Number),
+							Owner:       monitoring.ObservableOwnerCoreApplication,
+							Interpretation: `
+								Indicates how many repositories have been scheduled for a permissions sync.
+							`,
```
Comment (bobheadxi) [inline monitoring/definitions/repo_updater.go:313]: Thanks for adding an interpretation 🙏 Could we add a link to docs about what permissions sync is, and what scheduling means in this context? There's some relevant sections here: https://docs.sourcegraph.com/admin/repo/permissions#background-permissions-syncing

Comment (bobheadxi) [reply monitoring/definitions/repo_updater.go:313]: In general, when writing an interpretation or possible solutions, try to leave a path to learn more about what an observable means for someone who has no context on how Sourcegraph works

Comment (sashaostrikov) [reply monitoring/definitions/repo_updater.go:313]: sure! thanks for noticing, I've just done it [here](https://github.com/sourcegraph/sourcegraph/pull/28337) as a follow-up to this PR

**Pull request merged**